### ISSUE TITLE
Undescriptive error when parsing

### ISSUE THREAD

**Issue by matdurand** [zarldev/goenums#25]
Hi,

I encountered some situation while parsing JSON and it took me a few minutes to track down that the error was coming from the `ParseXXX` method, more specifically
```
return res, fmt.Errorf("failed to parse %v", a)
```

What if we change that message to something a bit more explicit like
```
return res, fmt.Errorf("failed to parse %v into a valid XXX", a)
```
where XXX would be the enum name.

**Comment by zarldev** [comment]
Hey @matdurand was looking in the code and can't find that string - 

the only occurrence I can see is : https://github.com/zarldev/goenums/blob/f41660bc0bbebe5bc4f5cd4d33b19b6b21362158/pkg/generator/generator.go#L651

**Comment by matdurand** [comment]
Nevermind. I'm using 0.3.4 and this has been fixed in 0.3.5
https://github.com/zarldev/goenums/compare/v0.3.4...v0.3.5#diff-9fde1b602271222d4351f8e635e3398b84f0af0cc0df19ce8b9f5095fefb5c1bL651
